README.md

```diff
@@ -289,3 +290,75 @@ geth console --datadir <datadir>
 ```
 
 It seems that this issue is caused by the celo-blockchain client sometimes shutting down in an inconsistent state, which is repaired upon the next startup.
+
+## Running a challenger
+
+> In principle, running a challenger is essentially the same as operating an L2 node, with an additional service that compares the proposed dispute game L2 state root with its locally synced L2 state. Whenever the proposed state root does not match the operator's local state, the challenger publishes a challenge to L1 and the proposer is required to compute a zk-proof for the proposed state root.
+> This mechanism requires the challenger operator to run as much of the L2 infrastructure as possible locally and to avoid relying on third-party services without independently deriving consensus.
+> Because the state root must be compared to historical state roots up to approximately one week old, the operator must run a local L2 archive node.
+> In order to ensure the ability to fully derive the L2 state from consensus L1 data, the challenger operator should also run a dedicated EigenDA proxy service rather than connecting to a public S3-backed batch cache.
+
+## Installation and Configuration
+
+Refer to the [previous instructions](#installation-and-configuration) and the [Celo Docs](https://docs.celo.org/cel2/operators/run-node) on how to run a node and configure it as an archive node.
+It is possible to run the archive node with snap sync instead of full sync; however, in that case you must wait approximately one week before you can verify the validity of older but still active games.
+During this period, the challenger will observe errors of the form `missing trie node ... state is not available`.
+
+### Required steps
+
+Modify the `.env` file that you copied over with the following options:
+
+```
+NODE_TYPE=archive
+OP_GETH__SYNCMODE=full
+DATADIR_PATH=<path to a migrated L1 full node datadir>
+
+# disable fetching blobs from external (or local) cache
+EIGENDA_LOCAL_ARCHIVE_BLOBS=
+
+CHALLENGER__ENABLED=true
+```
+
+### Optional changes
+
+```
+# Reduces load on the L1 node. Since game timeouts are on the order of days, this is generally acceptable from a network security perspective.
+# Note, however, that submitting challenges is on a first-come, first-served basis. Configuring a higher value than other
+# operators will significantly reduce the likelihood of being the one to submit the challenge.
+# Conversely, lowering this value increases the probability of successfully submitting a challenge, but also increases load on the L1 node.
+CHALLENGER__FETCH_INTERVAL_SECONDS=120
+
+# Use a specific version of the challenger image
+IMAGE_TAG__CHALLENGER=v1.0.0
```

Comment:
Would be super helpful to keep always most recent tag for mainnet here (maybe url to https://github.com/celo-org/op-succinct/releases ?)

Reply:
I would say this is not strictly required, since this flag is an optional overwrite and just an example.

But for the default in docker-compose this is something that we have to add after we cut the non-rc release.
There will be the `v1` tag then, and this should be the default.